vpd-manager/src/parser.cpp

```diff
@@ -198,6 +205,12 @@ int Parser::updateVpdKeyword(const types::WriteVpdParams& i_paramsToWriteData)
         l_bytesUpdatedOnHardware = constants::FAILURE;
     }
 
+    // Disable Reboot Guard
+    if (constants::FAILURE == dbusUtility::DisableRebootGuard())
+    {
+        // TODO: Log a Critical PEL here
```

Comment:
Log a PEL, lets avoid TODOs now. The API is already there.
Critical PEL should be logged calling out firmware.